src/workerd/io/actor-cache.c++

```diff
@@ -12,6 +12,10 @@
 #include <workerd/io/io-gate.h>
 #include <workerd/util/sentry.h>
 
+// TODO(cleanup): Break dependency on IoContext. ActorCache is intended to be independent of the
```

Comment:
no action: Good to know, we'll keep that in mind for future work.

Reply:
FWIW this comment is copy/pasted from later in the file, not actually new.